### PR TITLE
Install libfuse3 in jenkins images

### PR DESCRIPTION
**Pull request by ssz1997**
The jenkins image didn't install libfuse3 packages, which means in github actions the Fuse integration tests never ran with libfuse3. 

Tested on https://github.com/ssz1997/alluxio/pull/2 and the Fuse integration tests passed: https://github.com/ssz1997/alluxio/runs/6293974135?check_suite_focus=true

### CODE REVIEW

**Comment by ssz1997** [comment]
@Xenorith @LuQQiu PTAL. Thanks.

**Review by Xenorith — APPROVED** [review]

File: dev/jenkins/Dockerfile-jdk11 [hunk 1]
```diff
@@ -105,7 +105,7 @@ RUN mkdir -p /home/jenkins && \
     chmod -R 777 /.config && \
     apt-get update -y && \
     apt-get upgrade -y ca-certificates && \
-    apt-get install -y build-essential fuse libfuse-dev make ruby ruby-dev
+    apt-get install -y build-essential fuse libfuse-dev libfuse3-3 libfuse3-dev make ruby ruby-dev
```
Comment (LuQQiu) [inline dev/jenkins/Dockerfile-jdk11:108]: why for fuse 2 : fuse, libfuse-dev are installed
but for fuse 3: libfuse3-3 libfuse3-dev are installed

can fuse library works for both libfuse 2 & 3
why we don't need to install libfuse package?

Comment (ssz1997) [reply dev/jenkins/Dockerfile-jdk11:108]: Thanks for spotting this. I didn't notice package `fuse3` exists and thought `libfuse3-3` is the counterpart of `fuse`. It should be replaced by `fuse3`. `fuse3` contains `libfuse3-3` and a few other basic packages.

I'm assuming you are talking about fuse kernel module. Although some functionalities require higher kernel version, from my testing AlluxioFuse works without any problems.
Both `fuse` and `fuse3` contain libfuse packages.

**Review by LuQQiu — APPROVED** [review]

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**